packages/mongo-nav/src/MongoNav.spec.tsx

```diff
@@ -92,63 +56,64 @@ describe('packages/mongo-nav', () => {
     });
 
     test('the organization and project navs are rendered', () => {
-      expect(expectedElements.orgNav).toBeInTheDocument();
-      expect(expectedElements.projectNav).toBeInTheDocument();
+      expect(queryByTestId('organization-nav')).not.toBeNull();
```

Comment:
Since now we're querying the element, it must be in the document if it's not `null`. Before the element could've been stale and no longer in the document. Same rationale as https://github.com/mongodb/leafygreen-ui/pull/465/files#r455182402.

Reply:
What's the difference in log if we made this intentionally fail?

```jsx
expect(queryByTestId('organization-nav')).toBeNull();
// vs
expect(queryByTestId('organization-nav')).toBeInTheDocument();
```

Also what does the query log error show when it errors?

Reply:
the former gives us
```
expect(received).not.toBeNull()

Received: null

      58 |     test.only('the organization and project navs are rendered', () => {
    > 59 |       expect(queryByTestId('doesnt-exist')).not.toBeNull();
```
the latter gives us
```
expect(received).toBeInTheDocument()

    received value must be an HTMLElement or an SVGElement.
    Received has value: null

      58 |     test.only('the organization and project navs are rendered', () => {
    > 59 |       expect(queryByTestId('doesnt-exist')).toBeInTheDocument();
```

The latter adds some extra context about the expected type, but since you can see the line it failed on, it should be pretty obvious either way what we were expecting.

Reply:
Yeah, the latter definitely seems more accurate to me. We want to make sure that it's in the document and visible (which we're not strictly testing for visibility, but in document is closer than just not null).